src/components/Article.jsx

```diff
@@ -0,0 +1,95 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import formatDate from "../assets/utils";
+
+function Article({ users }) {
+  const { article_id } = useParams();
+  const [article, setArticle] = useState(null);
+  const [comments, setComments] = useState([]);
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    setLoading(true);
+
+    const fetchArticle = axios.get(
+      `https://news-api-project-hj1l.onrender.com/api/articles/${article_id}`
+    );
+
+    const fetchComments = axios.get(
+      `https://news-api-project-hj1l.onrender.com/api/articles/${article_id}/comments`
+    );
+
+    Promise.all([fetchArticle, fetchComments]).then((response) => {
```

Comment:
i would be wary of using a promise.all here - the user has to wait for both requests to complete before they see anything. Another way to do this is the have a separate loading state for each request and display what ever data comes back as soon as it comes back. if one fails, they both fail here and nothing gets shown to the user!